plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/StandardProjectWizard.java

```diff
@@ -1,28 +1,32 @@
 package com.google.cloud.tools.eclipse.appengine.newproject;
 
 import org.eclipse.jface.viewers.IStructuredSelection;
-import org.eclipse.jface.wizard.IWizardPage;
 import org.eclipse.jface.wizard.Wizard;
 import org.eclipse.ui.INewWizard;
 import org.eclipse.ui.IWorkbench;
 
 public class StandardProjectWizard extends Wizard implements INewWizard {
 
+  // visible for test
+  AppEngineStandardWizardPage page;
+
   @Override
   public void init(IWorkbench workbench, IStructuredSelection selection) {
     this.setWindowTitle("New App Engine Standard Project");
-    // see BasicNewResourceProjectWizard which is what we need except it isn't meant to be subclassed
-    IWizardPage page = new StubWizardPage("first page");
+    page = new AppEngineStandardWizardPage("first page");
     this.addPage(page);
   }
   
   @Override
   public boolean canFinish() {
```

Comment:
Remove this method then — the default iterates over the pages to verify that they are all complete.